### DISCUSSION TITLE
hyprland-nvidia-git is better than hyprland-git for nvidia.

### DISCUSSION THREAD

**Discussion by 7thripr** [prasanthrangan/hyprdots#53] [General]
Hi @prasanthrangan,

I was facing some issue with hyprland on nvidia tried changing it to hyprland-nvidia and is flawless would it mind you to detect nvidia and install required drivers and use hyprland-nvidia instead would be great.

Thank you

**Comment by prasanthrangan** [comment]
It should already switch from `hyprland-git` to `hyprland-nvidia-git` if nvidia card is detected. Does it not detect nvidia card in your system?

**Reply by 7thripr** [reply]
It didnt work some weeks back i will try again. But was waiting for a major milestone in the script to backup and flash.
